platforms/src/ETH/Providers/accountAnalysis.ts

```diff
@@ -1,39 +1,47 @@
 // ----- Types
-import { ProviderExternalVerificationError, type Provider } from "../../types";
+import { type Provider } from "../../types";
 import type { RequestPayload, VerifiedPayload, ProviderContext, PROVIDER_ID } from "@gitcoin/passport-types";
 import axios from "axios";
+import { handleProviderAxiosError } from "../../utils/handleProviderAxiosError";
 
-type HumanProbability = {
-  human_probability: number;
-};
+type HumanProbability = number;
```

Comment:
Isn't it redundant ?

Reply:
Well it's defining that a HumanProbability will always be a number, and that's used in two spots. But we never do that anywhere else, so you're right that I should probably drop this!